src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -75,6 +75,19 @@ def test_help_version_arguments_output_and_exit(
                 ["--endpoint", "v1/chat/completions"],
                 {"endpoint": "v1/chat/completions"},
             ),
+            (
+                ["--synthetic-extra-inputs", "random_key:random_value"],
+                {"synthetic_extra_inputs": ["random_key:random_value"]},
+            ),
+            (
+                [
+                    "--synthetic-extra-inputs",
+                    "random_key:5",
+                    "--synthetic-extra-inputs",
+                    "another_random_key:6",
+                ],
+                {"synthetic_extra_inputs": ["random_key:5", "another_random_key:6"]},
```

Comment:
what happens if the same value is entered twice?

Reply:
The second one gets used, since dictionaries ignore repeated keys. I added exception handling to avoid unexpected behavior. I also added unit testing to confirm behavior.

Reply:
awesome. thank you